spec/spec_helper.rb

```diff
@@ -133,4 +133,24 @@
     example.run
     Timecop.return
   end
+
+  # enable `allow_forgery_protection` in Lighthouse specs to ensure that the endpoints
+  # in those modules have explicitly skipped the CSRF protection functionality
+  lighthouse_dirs = %r{
+    modules/
+    (appeals_api|claims_api|openid_auth|va_facilities|va_forms|vba_documents|
+      veteran|veteran_confirmation|veteran_verification)/
```

Comment:
Can this somehow be defined with a wildcard in the event that a module is added/modified/removed? Would be a serious pain to understand what the heck is going on here for some poor developer in the future.

Reply:
I agree it's not ideal. We are relying on any future Lighthouse modules being added manually too.
I'm open to suggestions.  There's nothing in there naming that identifies them as Lighthouse manuals.  And I can't just include everything in `modules`. `vaos`, for example, lives there too.